### PR TITLE
Feature/header reworking

### PR DESCRIPTION
**Pull request by SviatlanaHlukhava**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -55,48 +55,54 @@
...
+		            <li ng-controller="ideasSearchCtrl as ideasSearchCtrl" ng-show="ideasSearchCtrl.isSearchVisible()">
+		                <input placeholder="&#57347; {{ 'FIND' | translate }}..." ng-model="query" ng-change="ideasSearchCtrl.changeQuery()" class="form-control input-lg searcher glyphicon" type="search"/>
+		            </li>
+		            <li ng-if="authenticated"><a id="ideaAddNew" href="#ideaAddNew" translate="I_HAVE_IDEA"></a></li>
+		        </ul>
+
+		        <ul class="nav navbar-nav navbar-right">
+		            <li class="dropdown">
+		                <a id="username" ng-if="authenticated" href="" class="dropdown-toggle ng-binding" data-toggle="dropdown" aria-haspopup="true" >
+			                <i ng-if="currentUser.socialSignInProvider == null || currentUser.socialSignInProvider == 'NONE'" class="fa fa-user"></i><i ng-if="currentUser.socialSignInProvider === 'GOOGLE'" class="fa fa-google-plus-square color-google-plus"></i><i ng-if="currentUser.socialSignInProvider === 'FACEBOOK'" class="fa fa-facebook-square color-facebook"></i><i ng-if="currentUser.socialSignInProvider === 'VKONTAKTE'" class="fa fa-vk color-vk"></i>
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/index.html]: Please, move logic for selecting link class to controller (Google, Facebook, ...)
https://docs.angularjs.org/api/ng/directive/ngClass

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Comment by kmih** [comment]
:+1:

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Pull request merged**